src/renderer/components/dock/log.store.ts

```diff
@@ -65,72 +73,73 @@ export class LogStore {
    * @param tabId
    */
   loadMore = async (tabId: TabId) => {
-    if (!this.podLogs.get(tabId).length) return;
-    const oldLogs = this.podLogs.get(tabId);
-    const logs = await this.loadLogs(tabId, {
-      sinceTime: this.getLastSinceTime(tabId)
-    });
+    if (!this.podLogs.get(tabId).length) {
+      return;
+    }
+
+    try {
+      const oldLogs = this.podLogs.get(tabId);
+      const logs = await this.loadLogs(tabId, {
+        sinceTime: this.getLastSinceTime(tabId)
+      });
 
-    // Add newly received logs to bottom
-    this.podLogs.set(tabId, [...oldLogs, ...logs]);
+      // Add newly received logs to bottom
+      this.podLogs.set(tabId, [...oldLogs, ...logs]);
+    } catch (error) {
+      this.handlerError(tabId, error);
+    }
   };
 
   /**
    * Main logs loading function adds necessary data to payload and makes
    * an API request
    * @param tabId
    * @param params request parameters described in IPodLogsQuery interface
-   * @returns {Promise} A fetch request promise
+   * @returns A fetch request promise
    */
-  loadLogs = async (tabId: TabId, params: Partial<IPodLogsQuery>) => {
+  @autobind()
+  async loadLogs(tabId: TabId, params: Partial<IPodLogsQuery>): Promise<string[]> {
     const data = logTabStore.getData(tabId);
     const { selectedContainer, previous } = data;
     const pod = new Pod(data.selectedPod);
     const namespace = pod.getNs();
     const name = pod.getName();
 
-    return podsApi.getLogs({ namespace, name }, {
+    const result = await podsApi.getLogs({ namespace, name }, {
       ...params,
       timestamps: true,  // Always setting timestampt to separate old logs from new ones
       container: selectedContainer.name,
       previous
-    }).then(result => {
-      const logs = [...result.split("\n")]; // Transform them into array
-
-      logs.pop();  // Remove last empty element
-
-      return logs;
     });
-  };
+
+    return result.trimEnd().split("\n");
+  }
 
   /**
    * Converts logs into a string array
-   * @returns {number} Length of log lines
+   * @returns Length of log lines
    */
```

Comment:
Another signal that such comments should be avoided in overall. Once described they tempted to get away from code and started to provide lies. Moreover, this description was invalid right from the start.

The function code was changed, but description comment left untouched.

Reply:
I *emphatically* do not agree that we should avoid comments because the code might be updated.

We should as a team require in code reviews that all comments should be updated or at least marked as checked whenever we change the functions. This codebase is slowly getting larger and we should leave helpful guides to future maintainers about what the code does.